tests/upgrade_path/test_upgrade_path.py

```diff
@@ -161,6 +158,6 @@ def test_upgrade_path(localhost, duthost, ptfhost, upgrade_path_lists, ptf_param
                        platform_dir="ptftests",
                        params=test_params,
                        platform="remote",
-                       qlen=1000,
+                       qlen=10000,
```

Comment:
If `qlen=10000` is found to be better setting, should we also change `PTFRUNNER_QLEN` to 10000 in advanced_reboot.py script too?

Reply:
I'm not sure about that. We need to run test_advanced_reboot and check the ptf log. There will be a debug log in ptf log
```Discarding oldest packet to make room```